src/routes/Settings/utils/index.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+import formatBoolean from './formatBoolean'
+import formatCurrency from './formatCurrency'
+import formatLanguage from './formatLanguage'
+
+export {
+  formatBoolean,
+  formatCurrency,
+  formatLanguage,
+}
```

Comment:
need to move these settings utils to `src/utils`

Reply:
This functions using only for Settings module now, I not sure that we need to move them to global namespace